library/src/main/java/com/novoda/downloadmanager/lib/DownloadStatus.java

```diff
@@ -13,6 +13,10 @@
 final class DownloadStatus {
 
     /**
+     * This download has been marked for deletion and it will be deleted in the future
+     */
+    public static final int DELETING = 188;
```

Comment:
yeah but status codes between 400 and 500 count as an error - just wondering if this is an error but probably not